.travis.yml

```diff
@@ -1,10 +1,9 @@
 language: php
 
 php:
-  - 5.4
-  - 5.5
-  - 5.6
   - 7.0
+  - 7.1
+  - 7.2
   - hhvm
```

Comment:
I think you can remove hhvm, as Symfony do not support it either :)

Reply:
LOL... removed, and then added back. :)